client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MlClientDocumentationIT.java

```diff
@@ -3112,6 +3112,10 @@ public void testStopDataFrameAnalytics() throws Exception {
             StopDataFrameAnalyticsRequest request = new StopDataFrameAnalyticsRequest("my-analytics-config"); // <1>
             // end::stop-data-frame-analytics-request
 
+            //tag::stop-data-frame-analytics-request-force
+            request.setForce(false); // <2>
```

Comment:
Since this is in a different doc tag callout, I think the doc build will fail. You may need to move 
```
request.setForce(false); // <2>
```
Up in side the `// tag::stop-data-frame-analytics-request` tag